src/lib.rs

```diff
@@ -31,9 +31,9 @@
 #![cfg_attr(all(test, feature = "unstable"), feature(test))]
 #[cfg(all(test, feature = "unstable"))] extern crate test;
 
-#[cfg(any(test, feature="std"))] extern crate core;
-#[cfg(feature="serde")] extern crate serde;
-#[cfg(all(test,feature="serde"))] extern crate serde_test;
+#[cfg(any(test, feature="std"))] pub extern crate core;
```

Comment:
Does it make sense to export core? It's not a third party dep, right?

Reply:
You are right, it can be removed. Done

Reply:
No, I have to bring it back, otherwise the hash_newtype results in 
```
error[E0603]: crate `core` is private
 --> <::hashes::util::hex_fmt_impl macros>:2:18
```